app/models/manageiq/providers/azure/refresh_helper_methods.rb

```diff
@@ -61,13 +61,13 @@ def gather_data_for_this_region(arm_service, method_name = 'list_all')
   # the resource group they live in, we do not filter by region here.
   #
   def resource_groups
-    @resource_groups ||= @rgs.list
+    @resource_groups ||= @rgs.list(:all => true)
   end
 
   # Given an object, return the matching ems_ref for its resource group.
   #
   def get_resource_group_ems_ref(object)
-    "/subscriptions/#{object.subscription_id}/resourceGroups/#{object.resource_group.downcase}"
+    "/subscriptions/#{object.subscription_id}/resourceGroups/#{object.resource_group}".downcase
```

Comment:
you might as well just change the G in resourceGroups manually, cause it will never be capitalized in the final string :)

Reply:
I suppose that's true.